src/features/registry.js

```diff
@@ -58,4 +58,20 @@ systemJSPrototype.delete = function (id) {
         depLoad.i.splice(importerIndex, 1);
     });
   return delete this[REGISTRY][id];
-};
\ No newline at end of file
+};
+
+systemJSPrototype[Symbol && Symbol.iterator ? Symbol.iterator : '@@iterator'] = function() {
```

Comment:
Note the `Symbol` check is missing a `typeof Symbol` here.

I would prefer we only set the `Symbol.iterator` property when `Symbol` is defined, and we likely don't even need to do the `Symbol.iterator` check as I believe all browsers supporting symbols will support `Symbol.iterator`?